src/services/agile/providers/azureDevOps/azureDevOpsAgileService.integration.test.ts

```diff
@@ -32,7 +32,7 @@ describe("Azure DevOps Backlog Service", () => {
     providerOptions,
   });
 
-  it("can create sprints", async () => {
+  it("can create, get and delete sprints", async () => {
```

Comment:
Tests are cheap: Why not split up instantiation into a before or a private method and then have one test for each of these things?

Reply:
Mostly to maintain atomicity of the test - I wanted a test that had no dependencies on previously existing work items and that could clean up after itself. This one creates a new work item, verifies that it indeed exists (by using the `get` and checking that it's equal to what we expect) and then cleans up after itself (verifying that it cleaned up by running another `get` and expecting failure). Open to other thoughts. I just thought having one all-inclusive test would be helpful to test the various parts